codeSnippets/snippets/full-stack-task-manager/server/src/main/kotlin/com/example/ktor/full_stack_task_manager/Application.kt

```
@@ -0,0 +1,92 @@
+                }
+                try {
+                    val priority = Priority.valueOf(priorityAsText)
+                    val tasks = repository.tasksByPriority(priority)
+
+
+                    if (tasks.isEmpty()) {
+                        call.respond(HttpStatusCode.NotFound)
+                        return@get
+                    }
+                    call.respond(tasks)
+                } catch (ex: IllegalArgumentException) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)